packages/default-theme/components/SwProductDetails.vue

```diff
@@ -1,5 +1,7 @@
 <template>
   <div class="sw-product-details">
+    <SwTipPopup />
```

Comment:
here should be v-if isMobile and component should be lazy-loaded only in mobile

Reply:
Added isMobile here